models/topicsModel.js

```diff
@@ -1,9 +1,16 @@
 const db = require("../db/connection");
+const fs = require("fs/promises");
 
 selectTopics = () => {
   return db.query(`SELECT * FROM topics`).then((topics) => {
     return topics.rows;
   });
 };
 
-module.exports = { selectTopics };
+const selectAPI = () => {
+  return fs.readFile(__dirname + "/../endpoints.json", "utf-8").then((data) => {
+    return JSON.parse(data);
+  });
+};
```

Comment:
While this is working, consider the operations here. 
- reading a json to a string
- parsing that string back to an object
- sending that object to a controller
- the controller sends back a json object to the client.
Could that journey be made shorter?